libraries/abstractions/ble_hal/include/bt_hal_manager_adapter_ble.h

```diff
@@ -97,8 +97,15 @@ typedef struct
     BTGattAdvName_t ucName;
     bool bSetScanRsp;
     uint32_t ulAppearance;
-    uint32_t ulMinInterval; /**< Minimum Connection Interval. If set to 0, minimum connection interval is not included in advertisement/scan response data. */
-    uint32_t ulMaxInterval; /**< Maximum Connection Interval. If set to 0, maximum connection interval is not included in advertisement/scan response data. */
+    uint32_t ulMinInterval;    /**< Minimum Connection Interval. If set to 0, minimum connection interval is not included in advertisement/scan response data. */
+    uint32_t ulMaxInterval;    /**< Maximum Connection Interval. If set to 0, maximum connection interval is not included in advertisement/scan response data. */
+    uint16_t usMinAdvInterval; /**< Minimum Advertising Interval in units of 0.625ms.
+                                *   Range: 0x0020 to 0x4000. Time Range: 20 ms to 10.24 s.
```

Comment:
Should the min and max range values be provided as #defines?

Reply:
Min and max range are  the highest and lowest possible values for the intervals and is validated internally by  stack. Typical user values will be much narrower that this range. Comment here is an indication that user should provide values within this range.